vscode-dotnet-runtime-library/src/Acquisition/AcquisitionInvoker.ts

```diff
@@ -110,29 +113,60 @@ You will need to restart VS Code after these changes. If PowerShell is still not
         }
     }
 
+    private TryFindWorkingCommand(commands : string[]) : [string, boolean]

```

Comment:
for this pattern you tend to find a return type more like this:

`: { success: true, command: string } | { success : false }`

then, you can check `if (result.success)` and inside that check the compiler will know that you have access to the `command` property. This is a lot clearer than a typescript tuple (aka 2-element array).

Reply:
Thank you, I will make this change in the next PR because I would like to release and this seems more prone to me making an error (vendors approved this one)